apps/web/vibes/soul/sections/address-list-section/index.tsx

```diff
@@ -342,12 +357,18 @@ function AddressActionButton({
       <input {...getInputProps(fields.lastName, { type: 'hidden' })} key={fields.lastName.id} />
       <input {...getInputProps(fields.company, { type: 'hidden' })} key={fields.company.id} />
       <input {...getInputProps(fields.phone, { type: 'hidden' })} key={fields.phone.id} />
-      <input {...getInputProps(fields.street1, { type: 'hidden' })} key={fields.street1.id} />
-      <input {...getInputProps(fields.street2, { type: 'hidden' })} key={fields.street2.id} />
+      <input {...getInputProps(fields.address1, { type: 'hidden' })} key={fields.address1.id} />
+      <input {...getInputProps(fields.address2, { type: 'hidden' })} key={fields.address2.id} />
       <input {...getInputProps(fields.city, { type: 'hidden' })} key={fields.city.id} />
-      <input {...getInputProps(fields.state, { type: 'hidden' })} key={fields.state.id} />
+      <input
+        {...getInputProps(fields.stateOrProvince, { type: 'hidden' })}
+        key={fields.stateOrProvince.id}
+      />
       <input {...getInputProps(fields.postalCode, { type: 'hidden' })} key={fields.postalCode.id} />
-      <input {...getInputProps(fields.country, { type: 'hidden' })} key={fields.country.id} />
+      <input
+        {...getInputProps(fields.countryCode, { type: 'hidden' })}
+        key={fields.countryCode.id}
```

Comment:
What do we need all these hidden inputs for? Aren't there actual inputs that have these values?

Reply:
For the delete and set default buttons, we don't have a form, but we want to update the optimistic state here.

Reply:
I see. Feels a bit weird to have to do this but I don't have any solutions to offer right now so carry on 😅